registrations/utils.py

```diff
@@ -1,6 +1,31 @@
 import phonenumbers
+from django.conf import settings
+from django.core.cache import caches
+from temba_client.v2 import TembaClient
+
+CACHE = caches["contacts"]
+CACHE_EXPIRY_SECONDS = 600
 
 
 def normalise_msisdn(msisdn):
     msisdn = phonenumbers.parse(msisdn, "ZA")
     return phonenumbers.format_number(msisdn, phonenumbers.PhoneNumberFormat.E164)
+
+
+def get_contact(msisdn):
```

Comment:
Maybe we should have `rapidpro` somewhere in these function names, since WhatsApp also has the concept of a `contact`, it could be confusing